google/cloud/storage/internal/crc32c_benchmark.cc

```diff
@@ -0,0 +1,99 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/storage/internal/crc32c.h"
+#include <benchmark/benchmark.h>
+#include <crc32c/crc32c.h>
+
+namespace google {
+namespace cloud {
+namespace storage_internal {
+GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
+namespace {
+
+// Run on (128 X 2250 MHz CPU s)
+// CPU Caches:
+//   L1 Data 32 KiB (x64)
+//   L1 Instruction 32 KiB (x64)
+//   L2 Unified 512 KiB (x64)
+//   L3 Unified 16384 KiB (x16)
+// Load Average: 1.88, 2.61, 6.87
+// ----------------------------------------------------------------------
+// Benchmark                            Time             CPU   Iterations
+// ----------------------------------------------------------------------
+// BM_Crc32cDuplicateNonAbseil   25520759 ns     25520833 ns           28
+// BM_Crc32cDuplicate            24168074 ns     24168122 ns           28
```

Comment:
Now that I see the implementation, I am not surprised that they take 2x as long.

Reply:
Ack.